org.eclipse.xtext.util/build.gradle

```diff
@@ -5,3 +5,9 @@ dependencies {
 	compile "org.eclipse.emf:org.eclipse.emf.ecore:$versions.emfEcore"
 	compile "com.google.inject:guice:$versions.guice"
 }
+
+jar {
+	from('.') {
+		include 'plugin.properties'
```

Comment:
Please move this configuration to `eclipse-project-layout.gradle`. If a plugin.properties is present, it should always be included in the build.